exporter/prometheusremotewriteexporter/README.md

```diff
@@ -48,6 +48,10 @@ Example:
 exporters:
   prometheusremotewrite:
     endpoint: "https://my-cortex:7900/api/v1/push"
+    wal: # Enabling the Write-Ahead-Log for the exporter.
+        directory: ./prom_rw # The directory to store the WAL in
+        buffer_size: 100 # Option count of elements to be read from the WAL before truncating; default of 300
```

Comment:
```suggestion
        buffer_size: 100 # Optional count of elements to be read from the WAL before truncating; default of 300
```

Reply:
Thank you @tigrannajaryan! I've addressed it.